ios/FluentUI/SegmentedControl/SegmentPillButton.swift

```diff
@@ -36,6 +36,12 @@ class SegmentPillButton: UIButton {
             var configuration = UIButton.Configuration.plain()
             configuration.contentInsets = Constants.insets
             configuration.background.backgroundColor = .clear
+            let titleTransformer = UIConfigurationTextAttributesTransformer { incoming in
```

Comment:
do we need to update segmented control's mask layer how it sets "            label.font = button.titleLabel?.font"  ?

Reply:
Doesn't seem like it. It should be getting the font after it goes through the transformer, so they should match.